### DISCUSSION TITLE
请问如何能获取到最新版QQ的协议信息

### DISCUSSION THREAD

**Discussion by xxyy3130** [Mrs4s/go-cqhttp#2388] [Q&A]
比如我想获得最新版QQ的下面这些信息🥺
{
    "apk_id": "com.tencent.mobileqq",
    "app_id": 537161402,
    "sub_app_id": 537161402,
    "app_key": "0S200MNJT807V3GE",
    "sort_version_name": "8.9.58.11170",
    "build_time": 1684467300,
    "apk_sign": "a6b745bf24a2c277527716f6f36eb68d",
    "sdk_version": "6.0.0.2545",
    "sso_version": 20,
    "misc_bitmap": 150470524,
    "main_sig_map": 34869472,
    "sub_sig_map": 66560,
    "dump_time": 1683193286,
    "protocol_type": 6
}

**Comment by 1umine** [comment]
https://github.com/RomiChan/protocol-versions/pull/6
